sympy/integrals/manualintegrate.py

```diff
@@ -1153,7 +1163,17 @@ def eval_derivativerule(integrand, symbol):
     if len(integrand.args) == 2:
         return integrand.args[0]
     else:
-        return sympy.Derivative(integrand.args[0], *integrand.args[1:-1])
+        # we are removing the last occurence of symbol
+        # from the list of differentiating variables
+        diff_vars = tuple(reversed(integrand.args[1:]))
+        symbol_index = diff_vars.index(symbol)
+        diff_vars_symbol_removed = diff_vars[:symbol_index] + diff_vars[symbol_index + 1:]
```

Comment:
You should be able to use `diff_vars.remove(symbol)` (if `diff_vars` is a list).